spoc/snipping/snipping_strategies.py

```diff
@@ -0,0 +1,237 @@
+"""Snipping strategies for Snipper that implement specific snipping functionality"""
+from abc import ABC, abstractmethod
+from functools import partial
+from enum import Enum
+from typing import Union
+from multiprocess.pool import ThreadPool
+import pandas as pd
+import numpy as np
+from sparse import COO
+import duckdb
+import bioframe
+from spoc.pixels import PersistedPixels
+
+
+class SnippingValues(Enum):
```

Comment:
Using Enum looks semantic, but will it be understandable for end users? I rarely encountered Enums in my work.

Reply:
Yes, good point. I can move it into the method and implement a way to instantiate the Enum with a string.